benchmarks/step_detect.py

```diff
@@ -17,4 +17,4 @@ def time_detect_regressions(self):
             step_detect.detect_regressions(self.y)
 
     def time_solve_potts_approx(self):
-        step_detect.solve_potts_approx(self.y, 0.3, p=1)
```

Comment:
Should be `gamma=0.3`. The option for setting `p` was removed.

Reply:
This is fixed